File: sbe/binance/spot_3_2/generated-php/spot_sbe/TimeInForce.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class TimeInForce
{
    public const GTC = 0;
    public const IOC = 1;
    public const FOK = 2;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}

File: sbe/binance/spot_3_2/generated-python/spot_sbe/TimeInForce.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple, Set
from io import BytesIO

class TimeInForce:
    """Enum values."""
    GTC = 0
    IOC = 1
    FOK = 2
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1
